backend/src/main/scala/sbt/internal/inc/bloop/internal/BloopAnalysisCallback.scala

```diff
@@ -68,8 +68,8 @@ final class BloopAnalysisCallback(
   private[this] val classPublicNameHashes = new mutable.HashMap[String, Array[NameHash]]
   private[this] val objectPublicNameHashes = new mutable.HashMap[String, Array[NameHash]]
   private[this] val usedNames = new mutable.HashMap[String, mutable.HashSet[UsedName]]
-  private[this] val unreporteds = new mutable.HashMap[File, mutable.ListBuffer[Problem]]
-  private[this] val reporteds = new mutable.HashMap[File, mutable.ListBuffer[Problem]]
+  private[this] val unreportedProblems = new mutable.HashMap[File, mutable.ListBuffer[Problem]]
```

Comment:
I'm not terribly certain about this change set. I could be wrong about what the objects are.

Reply:
Yeah, this is fine :+1: